### PR TITLE
Update notices.yml

### PR DESCRIPTION
**Pull request by gmauro**
Downtime message for the 20.05 version update.

### CODE REVIEW

**Comment by gmauro** [comment]
ping @bgruening

File: _data/notices.yml [hunk 1]
```diff
@@ -7,7 +7,13 @@
...
 #
 
-
+- title: Possible downtime on the 29.07.2020 
+  class: warning # one of [success, info, warning, danger]
+  site: freiburg # Your site
+  messages:
+      - message: |
+           On July 29th, we plan to update Usegalaxy.eu's codebase to the [20.05 Galaxy release](https://docs.galaxyproject.org/en/master/releases/20.05_announce_user.html). 
+           We expect a short downtime during the day and it is possible running jobs could be interrupted. Please take it into account in your job schedule.
```
Comment (beatrizserrano) [inline _data/notices.yml:16]:
```suggestion
           We expect a short downtime during the day and running jobs might be interrupted. Please take it into account in your job schedule.
```

File: _data/notices.yml [hunk 2]
```diff
@@ -7,7 +7,13 @@
 #           Here you can write some message for your users, [markdown](#asdf) is permitted.
 #
 
-
+- title: Possible downtime on the 29.07.2020
```
Comment (bgruening) [inline _data/notices.yml:10]: should we remove the possible ...

Comment (gmauro) [reply _data/notices.yml]: Better

**Pull request merged**